Assets/PatchKit Patcher/Scripts/AppUpdater/Commands/DownloadPackageCommand.cs

```diff
@@ -73,7 +74,11 @@ public override void Execute(CancellationToken cancellationToken)
 
             downloader.DownloadProgressChanged += bytes => { _status.Bytes.Value = bytes; };
 
-            downloader.Download(cancellationToken);
+            using (_status.BytesPerSecond.Subscribe(bps =>
+                _status.Description.Value = bps > 0 ? "Downloading package..." : "Stalled..."))
```

Comment:
I would use a small threshold while checking `bps` (like `0.01`). It's a `double` so we might encounter some floating point precision issues.